main/gy521_imu.h

```diff
@@ -79,5 +79,17 @@ void setupIMU(){
     Serial1.print("Z Acceleration: ");
     Serial1.println(sample.accel.z);
     delay(1000);
-  }
+    imuReading imuSample = getIMU();
```

Comment:
We already have an imuReading on line 67, so you can remove this line and use `sample` in all of the places that you used `imuSample`